test/common/index.js

```diff
@@ -812,7 +812,49 @@ function restoreWritable(name) {
   delete process[name].writeTimes;
 }
 
+function onResolvedOrRejected(promise, callback) {
```

Comment:
`finally` is a reserved word. I considered `promiseFinally` or `onFinally` - but decided to be explicit...